config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java

```diff
@@ -54,8 +54,8 @@
         boolean isBootstrap();
         boolean isFirstTimeDeployment();
         boolean useDedicatedNodeForLogserver();
-        boolean useFdispatchByDefault();
-        boolean dispatchWithProtobuf();
+        default boolean useFdispatchByDefault() { return false; } // TODO Remove once no longer used by old config models
```

Comment:
Usually we say "remove after Sept 2019". Alternatively, specify the version in the comment. We rely on people to remove those as they pass by because it is low effort ...

Reply:
I will update the comment to something more specific later. The method is still used by in the current config model. The motivation for this PR was to remove the feature flags and always enable the new dispatcher for hosted Vespa. We will cleanup the model code in the upcoming work package.